### PR TITLE
fix: do not redeclare variable

### PR DESCRIPTION
**Pull request by DanielRuf**
### Description of the Change
There is a variable redeclared in an example which would not work in some cases. In fact we do not have to redeclare this variable.

### Alternate Designs
N/A

### Why should this be in core?
N/A

### Benefits
N/A

### Possible Drawbacks
N/A

### Applicable issues
N/A

### CODE REVIEW

**Comment by craigtaub** [comment]
Thanks for the contribution.

**Comment by craigtaub** [comment]
Could you have a look at the failures? Then we can get this in, thanks

**Comment by DanielRuf** [comment]
https://travis-ci.org/mochajs/mocha/jobs/549688891#L471

> curl: (7) Failed to connect to nodejs.org port 443: No route to host

Not related to my changes as it seems.
Can you trigger the failing jobs to rerun?
Only people with the needed rights can do this.

**Comment by craigtaub** [comment]
Re-triggering seems to work but unfortunately can't do it for deploy preview and need that to be green. If you could rebase and push that would be really helpful please.

**Comment by DanielRuf** [comment]
I'll check later.

File: docs/example/Array.js [hunk 1]
```diff
@@ -70,4 +70,5 @@ describe('Array', function(){
       expect(arr).to.have.length(1);
     })
   })
-})
\ No newline at end of file
+})
+
```
Comment (outsideris) [inline docs/example/Array.js]:
```suggestion

```

**Pull request merged**